Release notes v2.11 (#2710)
openvinotoolkit/nncf: ReleaseNotes.md

# Release Notes

## New in Release 2.11.0

Post-training Quantization:

- Features:
  - (OpenVINO) Added Scale Estimation algorithm for 4-bit data-aware weights compression. The optional `scale_estimation` parameter was introduced to `nncf.compress_weights()` and can be used to minimize accuracy degradation of compressed models (note that this algorithm increases the compression time).

>   - (OpenVINO) Added GPTQ algorithm for 8/4-bit data-aware weights compression, supporting INT8, INT4, and NF4 data types. The optional `gptq` parameter was introduced to `nncf.compress_weights()` to enable the GPTQ algorithm.

Reviewer: add the reference to GPTQ: https://arxiv.org/abs/2210.17323

Author: Done, thanks.

>   - (OpenVINO) Added support for BF16 weights compression in `nncf.compress_weights()`.

Reviewer: Let's reformulate it to avoid any confusion (one can think that we compress models to bf16):
Added support for models with BF16 weights in the weights compression method, `nncf.compress_weights()`.

Author: Done, thanks.

>   - (PyTorch) Added support of the custom modules with traced parameters.

Reviewer 2: ```suggestion
  - (PyTorch) Added support for quantization and weight compression of the custom modules.
```

# Release Notes

## New in Release 2.11.0

Post-training Quantization:

- Breaking changes:
  - ...
- General:
  - ...
- Features:

>   - (OpenVINO) Added Scale Estimation algorithm for 4-bit data-aware weights compression. This algorithm estimates more optimal scale in compression scheme based on difference between matrix multiplication in original precision and compressed one. For that `scale_estimation` optional parameter has been added to `nncf.compress_weights()` and can be used to minimize accuracy degradation of compressed models (note that this option increases the compression time).

Reviewer 3: ```suggestion
  - (OpenVINO) Added Scale Estimation algorithm for 4-bit data-aware weights compression. This algorithm estimates a more optimal scale in compression scheme based on difference between output of matrix multiplication in the original precision and the compressed one. For that, the optional `scale_estimation` parameter has been added to `nncf.compress_weights()` and can be used to minimize accuracy degradation of compressed models (note that this option increases the compression time).
```

Author: Commit it, please.

Reviewer 3: @andreyanufr please confirm the change
